Cargo.toml

```diff
@@ -20,27 +20,31 @@ license.workspace = true
 [workspace.dependencies]
 serde = "1.0"
 lapce-xi-rope = { version = "0.3.2", features = ["serde"] }
-strum = "0.21.0"
-strum_macros = "0.21.1"
+strum = "0.26.2"
+strum_macros = "0.26.2"
 once_cell = "1.17.1"
+image = { version = "0.24", default-features = false, features = ["jpeg", "png"] }
```

Comment:
I think the defaults should be `"jpeg", "png", "ico", "webp", "gitf"` due to popularity but also being able to disable/enable any of them